src/main/java/adris/altoclef/butler/Butler.java

```diff
@@ -125,7 +125,7 @@ private void executeWhisper(String username, String message) {
         _commandFinished = false;
         _currentUser = username;
         sendWhisper("Command Executing: " + message, MessagePriority.TIMELY);
-        AltoClef.getCommandExecutor().execute("@" + message, () -> {
+        AltoClef.getCommandExecutor().execute(message, () -> {
```

Comment:
What is the intention here? Removing a prefix changes the functionality of Butler, which is beyond the scope of this PR, thus I need to see an explanation as to why this removal is necessary. (A functional change is one in which users would have to re-learn how to use the bot).

The point of a prefix is to make a declaration of intention. If a user wants to say "get wooden_sword 1", they can do so without triggering the bot. Only when they add an '@' will the bot understand the "intention" of the message. However, this is not needed in butler as whispering to the bot **is** a declaration of intention. As such, past builds of altoclef did not accept the prefix in the butler command. Users have grown accustomed to excluding this prefix, so we do not want to make a change that would require users to re-learn the butler system unless absolutely necessary.

(with the change made here, users would need to tell the bot "@get wooden_sword 1" instead of "get wooden_sword 1"


**Solution: Revert this change**